src/app/ui/AppProvider/ToastProvider.tsx

```diff
@@ -27,15 +39,70 @@ const Toast: FC<ToastProps> = ({ message }) => (
   </Box>
 );
 
-const ToastProvider: FC<PropsWithChildren> = ({ children }) => (
-  <RNTNToastProvider
-    offsetBottom={20}
-    renderToast={({ message }) => {
-      return <Toast message={message} />;
-    }}
-  >
-    {children}
-  </RNTNToastProvider>
-);
+const getDefaultBannerProps = (props: ToastProps) => {
+  let type = props.type?.replace('banner', '').toLowerCase(),
+    icon = props.icon,
+    bgColor = '';
+
+  switch (type) {
+    case 'success':
+      icon = <OcticonsCircleCheckFill color={colors.alertSuccess} size={24} />;
+      bgColor = colors.alertSuccessBg;
+      break;
+    case 'danger':
+      icon = <MaterialAlertOctagon color={colors.alertError} size={26} />;
+      bgColor = colors.alertErrorBg;
+      break;
+    default:
+      break;
+  }
+
+  return { icon, type, bgColor };
+};
+
+const Banner: FC<ToastProps> = (props: ToastProps) => {
+  const { top } = useSafeAreaInsets();
+  const { icon, bgColor } = getDefaultBannerProps(props);
+  return (
+    <Box
+      backgroundColor={bgColor}
+      width="100%"
+      py={40}
+      pt={isIOS ? top + 40 : 40}
```

Comment:
In general, I wouldn't assume that just because a device is not iOS, that it doesn't have a top safe area inset that should be taken into account. Do you know this for certain? I would expect that `useSafeAreaInsets()` returns an appropriate value for `top` for most Android devices, so I wouldn't expect to have to add another OS check here – but please correct me if I'm wrong, I haven't tested. 🙂

Reply:
As we discussed on Slack:

The banner is actually using the top safe area insets, this will place the banner normally, right down the status bar both ios/android as normally should be.

The mean of using `mt={IS_IOS ? -top : -12}` will put the banner at the top like if the status bar doesn't exists, but at the same time, Using the padding `pt={IS_IOS ? top + 12 : 12}`  will place the content on the right place bellow the status bar.

The intention is for IOS to remove the blue background behind the notch and use the same as the banner background, to have a smooth design.

![image](https://github.com/ChildMindInstitute/mindlogger-app-refactor/assets/3039328/5bc9d90f-bdd9-4a24-90e7-afa337e11393)

In android this is not needed since the status bar always remain black or with an accent color.